core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -462,7 +462,11 @@ object KafkaConfig {
     "the same security protocol to be usable in more than one port or IP. For example, we can separate internal and " +
     "external traffic even if SSL is required for both. Concretely, we could define listeners with names INTERNAL " +
     "and EXTERNAL and this property as: `INTERNAL:SSL,EXTERNAL:SSL`. As shown, key and value are separated by a colon " +
-    "and map entries are separated by commas. Each listener name should only appear once in the map."
+    "and map entries are separated by commas. Each listener name should only appear once in the map. " +
+    "We can provide different security (SSL and SASL) settings for each listener name by adding a normalised " +
```

Comment:
Can we please reword this not to use "we"? Generally, it's good to use a similar style as other configs.

Reply:
@ijuma  Updated the PR. Pls check.